Formula/tmux.rb

```diff
@@ -32,11 +34,16 @@ class Tmux < Formula
   def install
     system "sh", "autogen.sh" if build.head?
 
+    configure_args = [
+      "--disable-Dependency-tracking",
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+    ]
```

Comment:
I wasn't offended at all, and was trying to understand how important you felt the change was and whether I should go about changing it. I've hit a few projects where people are still not using the review system fully, so I think I'm still kinda of operating in the old model of GH reviews where a comment meant a change was requested unless otherwise specified.

I was more confused as to why I ran the style/convention checker, fixed issues before opening the PR, to then be told I've more style issues that need to be addressed. I was also trying to understand what I had missed, so that I could avoid needing more style fixes after opening a PR in the future.